src/app/components/hmcts-global-header/hmcts-global-header.component.spec.ts

```diff
@@ -93,7 +142,48 @@ describe('HmctsGlobalHeaderComponent', () => {
     expect(component.navigate.emit).toHaveBeenCalled();
   });
 
+  it('should display 16 digit case reference search box', async () => {
+    component.showItems = true;
+    component.showCaseReferenceSearchBox = true;
+    component.items = navItemsWithSearchBox;
+    await component.ngOnChanges(changesMock);
+    fixture.detectChanges();
+    const caseReferenceSearchBox = fixture.debugElement.query(By.css('#case-reference-search-box'));
+    expect(caseReferenceSearchBox).toBeTruthy();
+  });
+
+  it('should not display 16 digit case reference search box', async () => {
+    component.showItems = true;
+    component.showCaseReferenceSearchBox = false;
+    component.items = navItemsWithSearchBox;
+    await component.ngOnChanges(changesMock);
+    fixture.detectChanges();
+    const caseReferenceSearchBox = fixture.debugElement.query(By.css('#case-reference-search-box'));
+    expect(caseReferenceSearchBox).toBeFalsy();
+  });
+
+  it('should display find case right aligned', async () => {
```

Comment:
@johnbenjamin-hmcts - this unit test will need to be reworked following @ronaldmansveld's changes from PR #1396 because:

- The `showFindCase` flag is no longer used
- The `Find case` menu item no longer has an id of `find-case-link`; you'll need to change the CSS selector to `.hmcts-search-toggle__button` (i.e. relying on the class instead)

Reply:
Hi @johnbenjamin-hmcts - as agreed, I'll fix this test on the `feature/global-search` branch and then you can rebase your PR from it.